addon/components/frost-link.js

```diff
@@ -309,6 +325,18 @@ export default LinkComponent.extend(PropTypeMixin, HookMixin, SpreadMixin, {
   },
 
   /**
+   * Open a link for an external location.
+   */
+  _openExternalLink () {
+    const href = this.get('href')
+    // If we want to open a new tab, the target will be set accordingly when we setup the routing.
+    // If we want to stay in the same tab, we will set the href.
+    if (this._shouldOpenInSameTab()) {
+      window.location.href = href
```

Comment:
Looks like I never even opened a ticket with that defect, so I'll shut up now and let you do whatever you want with a component I have no use for anyway, since it doesn't actually affect me at all.